src/main/java/in/zapr/druid/druidry/granularity/DurationGranularity.java

```diff
@@ -34,19 +36,19 @@
     private final String type;
     @JsonProperty("duration")
     private long durationInMilliSeconds;
-    private DateTime origin;
+    private ZonedDateTime origin;
 
     public DurationGranularity(long durationInMilliSeconds) {
         this(durationInMilliSeconds, null);
     }
 
-    public DurationGranularity(long durationInMilliSeconds, DateTime origin) {
+    public DurationGranularity(long durationInMilliSeconds, ZonedDateTime origin) {
         this.type = DURATION_GRANULARITY_TYPE;
         this.durationInMilliSeconds = durationInMilliSeconds;
         this.origin = origin;
     }
 
     public String getOrigin() {
-        return origin == null ? null : origin.toDateTimeISO().toString();
+        return origin == null ? null : ISO_DATE_TIME.format(origin);
```

Comment:
I think we should use `ISO_OFFSET_DATE_TIME` formatter instead of `ISO_DATE_TIME`, as `ISO_DATE_TIME` formatter includes ZoneId along with offset.
Eg : Consider `origin` with ZoneId `Asia/Kolkata`
Then `ISO_DATE_TIME.format(origin)` will give you `2020-07-12T22:32:11.907+05:30[Asia/Kolkata]`
vs `origin.toDateTimeISO().toString()` will give you `2020-07-12T22:32:11.907+05:30`. 
I think using `ISO_DATE_TIME` will break current contract.